Source/Charts/Animation/Animator.swift

```diff
@@ -115,9 +103,9 @@ open class Animator: NSObject
                 elapsed = duration
             }
            
-            if _easingX != nil
+            if let _easingX = _easingX
```

Comment:
Why isn't this one line?
`phaseX = _easingX?(elapsed, duration) ?? Double(elapsed / duration)

Reply:
done